src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx

```diff
@@ -76,25 +97,28 @@ export class ArtworkSidebarBidAction extends React.Component<
     context_module: Schema.ContextModule.Sidebar,
     action_type: Schema.ActionType.Click,
   })
-  redirectToLiveBidding(user) {
+  redirectToLiveBidding(me: ArtworkSidebarBidAction_me | null) {
     const { slug } = this.props.artwork.sale
     const liveUrl = `${sd.PREDICTION_URL}/${slug}`
```

Comment:
We should use `getURL('PREDICTION_URL')` as that is SSR friendly.

Reply:
Addressed in 8c6fa53